s2nTests/docker-compose.yml

```diff
@@ -0,0 +1,62 @@
+version: '3'
+services:
+  saw-script:
+    build: ..
+    entrypoint: ["cp", "/usr/local/bin/saw", "/saw-bin"]
+    volumes:
+      - ./bin:/saw-bin:rw
+
+  hmac:
+    build:
+      context: .
+      args:
+        ARG_TEST: sawHMAC
+      dockerfile: docker/s2n.dockerfile
+    volumes:
+      - ./bin:/saw-bin:rw
+
+  drbg:
+    build:
+      context: .
+      args:
+        ARG_TEST: sawDRBG
+      dockerfile: docker/s2n.dockerfile
+    volumes:
+      - ./bin:/saw-bin:rw
+
+  sike:
+    build:
+      context: .
+      args:
+        ARG_TEST: sawSIKE
+      dockerfile: docker/s2n.dockerfile
+    volumes:
+      - ./bin:/saw-bin:rw
+
+  bike:
+    build:
+      context: .
+      args:
+        ARG_TEST: sawBIKE
+        ARG_LIBCRYPTO: openssl-1.0.2
+      dockerfile: docker/s2n.dockerfile
+    volumes:
+      - ./bin:/saw-bin:rw
+
+  tls:
+    build:
+      context: .
+      args:
+        ARG_TEST: tls
```

Comment:
these should be the same target, once `docker build` doesn't depend on `ARG_TEST`/`ARG_LIBCRYPTO`